.changeset/khaki-rockets-complain.md

```diff
@@ -0,0 +1,35 @@
+---
+"apollo-federation-integration-testsuite": minor
+"@apollo/query-planner": minor
+"@apollo/query-graphs": minor
+"@apollo/composition": minor
+"@apollo/federation-internals": minor
+"@apollo/gateway": minor
+---
+
+Introduce the new `@policy` scope for composition ([#2818](https://github.com/apollographql/federation/pull/2818))
```

Comment:
Nbd, this just isn't needed since changesets inserts the relevant PR into the changelog entry for us
```suggestion
Introduce the new `@policy` scope for composition
```